src/Evolution/Systems/ForceFree/RecoverPhysicalVariables.hpp

```diff
@@ -0,0 +1,90 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include "DataStructures/DataBox/Tag.hpp"
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Evolution/Systems/ForceFree/Tags.hpp"
+#include "PointwiseFunctions/GeneralRelativity/TagsDeclarations.hpp"
+#include "Utilities/TMPL.hpp"
+
+/// \cond
+class DataVector;
+namespace gsl {
+template <typename T>
+class not_null;
+}  // namespace gsl
+/// \endcond
+
+namespace ForceFree::Tags {
+/*!
+ * \brief Compute the electric field \f$E^i\f$ from TildeE.
```

Comment:
Did you know that we can do standard LaTeX syntax without the silly `\f`s these days? So just `$E^i$`. Also `\begin{equation}` etc. Not asking you to change anything, just FYI.

Reply:
oh great!